R/derive_params_growth_age.R

```diff
@@ -72,6 +72,12 @@
 #'  CDC developed extended percentiles (>95%) to monitor high BMI values,
 #'  if set to `TRUE` the CDC's correction is applied.
 #'
+#' @param who_correction Right skew correction
+#'
+#'  A logical scalar, e.g. `TRUE`/`FALSE` is expected.
+#'  WHO developed a modification to the z-score calculation to accommodate for right-skewness
+#'  in certain data, if set to `TRUE` the WHO correction is applied.
```

Comment:
I would replace line 78 and 79 text with the following to be more in line with the explanation from @CDC-DNPAO :

According to WHO guidelines for any weight-based indicator, to overcome right skewness in the data WHO decided to not rely on usual LMS method for extreme values and instead an extapolation approach is recommended.
More details on these exact rules applied can be found at the document page 302 of the [WHO Child Growth Standards Guidelines] (https://www.who.int/publications/i/item/924154693X).
If set to `TRUE` the WHO correction is applied.

Reply:
Zelos - i'll make a commit to update this and please check you agree

Reply:
@zdz2101 @rossfarrugia.  This SD23 WHO adjustment doesn't have anything to do with right skewness.  Actually, I can't even find 'right skewness' in the WHO 300+ page document.  Honestly, WHO did this because they decided to delete all kids with an abs z-score >3.  Then, they had to do something to calculate these extreme z-scores.  I would just refer to this as the  'WHO adjustment' or the WHO SD23 adjustment'.  

It's the L parameter in LMS that adjusts for right skewness using the Box-Cox transformation.   If I ever used the term 'right adjustment' for this, I apologize for my mistake

Reply:
I saw the phrase right-skewness somewhere on pg 302 before the formula, 
![image](https://github.com/user-attachments/assets/4948f9a3-bf0e-4a1c-8f3f-5015749f5878)


I'll remove any instances of the word "right skewnesss" and simply call it the `WHO adjustment` I don't think it's necessary for users to know the history of the decision made

Reply:
I agree that 'WHO adjustment' is much better